libs/blocks/merch-card/merch-card.css

```diff
@@ -1,5 +1,7 @@
 main > div[class*="-merch-card"],
-main > div[class*="-merch-cards"] {
+main > div[class*="-merch-cards"],
+.fragment > div[class*="-merch-card"],
+.fragment > div[class*="-merch-cards"] {
```

Comment:
Is there actually a need for both? `[class*="-merch-card"]` matches both `.something-merch-card` and `.something-merch-cards`, since both _contain_ `merch-card`. Am I missing something? This would make sense if instead of _contains_ (`*=`) you would use _ends with_ (`$=`), otherwise it could be trimmed a bit.